lib/quicly.c

```diff
@@ -104,6 +104,8 @@ KHASH_MAP_INIT_INT64(quicly_stream_t, quicly_stream_t *)
         quicly_escape_unsafe_string(alloca(_l * 4 + 1), (s), _l);                                                                  \
     })
 
+# define QUICLY_MIN(x, y) ((x) < (y) ? (x) : (y))
+
```

Comment:
I think we'd rather refrain from having this, considering that all other parts of quicly is written without.